### PR TITLE
Test indicators

### PR DESCRIPTION
**Pull request by mistergreen22**

### CODE REVIEW

**Comment by mistergreen22** [comment]
> Amazing!

please check tests on relationships to

File: tests/functional/tests/test_relationships.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+     ('domain', 'securecorp.club'),
+     ('sha256',
+      '9ffc7e4333d3be11b244d5f83b02ebcd194a671539f7faf1b5597d9209cc25c3'),
+     )
+)
+def test_positive_relationships(module_headers, observable, observable_type):
+    """Perform testing for enrich observe observables endpoint to check
+    relationships of Gigamon ThreatINSIGHT module
+
+    ID: CCTRI-1094-fb59a541-a42e-41fc-b859-52a1b564c14e
```
Comment (oshtaier) [inline tests/functional/tests/test_relationships.py:18]: copy/pasted =)

File: tests/functional/tests/test_indicators.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+    ID: CCTRI-1094-fb59a541-a42e-41fc-b859-52a1b564c14e
+
+    Steps:
+        1. Send request to enrich observe observable endpoint and check
+        indicators
+
+
+    Expectedresults:
+        1. Check that data in response body contains indicators entity
```
Comment (oshtaier) [inline tests/functional/tests/test_indicators.py:26]: Expected result usually mean result not action, so `Response body contains indicators entity with needed fields from  ThreatINSIGHT module` looks more valid

File: tests/functional/tests/test_relationships.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+    Steps:
+        1. Send request to enrich observe observable endpoint and check
```
Comment (oshtaier) [inline tests/functional/tests/test_relationships.py:21]: same

File: tests/functional/tests/test_relationships.py (hunk 1 above)
Comment (mistergreen22) [reply tests/functional/tests/test_relationships.py]: fixed

File: tests/functional/tests/test_indicators.py (hunk 2 above)
Comment (mistergreen22) [reply tests/functional/tests/test_indicators.py]: Done

File: tests/functional/tests/test_relationships.py (hunk 3 above)
Comment (mistergreen22) [reply tests/functional/tests/test_relationships.py:21]: If you mean same issue with expected result in this test case it's fixed, steps is little different(in one  - "check indicators", second - "check relationships")

**Pull request merged**